src/api/api.ts

```diff
@@ -261,7 +275,11 @@ function useIncompleteChecks(names?: string[]) {
     // Filter incomplete checks from the most recent ones
     return Array.from(checksByType.values())
       .filter((check) => (names ? names.includes(check.metadata.name ?? '') : true))
-      .filter((check) => !check.metadata.annotations?.[STATUS_ANNOTATION])
+      .filter(
+        (check) =>
+          !check.metadata.annotations?.[STATUS_ANNOTATION] ||
+          check.metadata.annotations?.[RETRY_ANNOTATION] !== undefined
```

Comment:
So we're intentionally treating `null` as a truthy value for `check.metadata.annotations?.[RETRY_ANNOTATION]`here, is that correct?

Reply:
yeah, well, the type of the variable is `string | undefined` so it should never be `null`, what I want here is to consider an empty string `''` truthy